### PR TITLE
Fix/workarounds

### PR DESCRIPTION
**Pull request by MP91**
# Description

## Azure DevOps PBI/Task reference

AB#_[PBI/Task number]_

## Checklist

* [ ] Vehicle App can be started with dapr run and is connecting to vehicle data broker
* [ ] Vehicle App can process MQTT messages and call the seat service
* [ ] Vehicle App can be deployed to local K3D and is running
* [ ] Created/updated tests, if necessary. Code Coverage percentage on new code shall be >= 70%.
* [ ] Extended the documentation in Velocitas repo
* [ ] Extended the documentation in README.md

* [ ] Devcontainer can be opened successfully
* [ ] Devcontainer can be opened successfully behind a corporate proxy
* [ ] Devcontainer can be re-built successfully

* [ ] Release workflow is passing

### CODE REVIEW

File: .vscode/tasks.json [hunk 1]
```diff
@@ -35,7 +37,8 @@
 			"group": "none",
 			"presentation": {
 				"close": true
-			}
+			},
```
Comment (kse3hi) [inline .vscode/tasks.json:40]: Can you please explain this change?

**Review by kse3hi — COMMENTED** [review]
LGTM

File: .vscode/tasks.json (hunk 1 above)
Comment (MP91) [reply .vscode/tasks.json:40]: this is just hiding this additional step of checking the output when launching the task

**Review by emsameen — APPROVED** [review]
LGTM

**Pull request merged**